EZSwiftExtensionsTests/UserDefaultTests.swift

```diff
@@ -25,5 +25,20 @@ class UserDefaultTests: XCTestCase {
         XCTAssertEqual(testUserDefaults?["foo"] as! String, "bar")
         XCTAssertNil(testUserDefaults?["wrongkey"])
     }
+    
+    func testDate() {
+        let testUserDefaults = UserDefaults(suiteName: "TEST")
+        let key = "dateTestKey"
+        let date: Date = Date()
```

Comment:
Following the [Type Interference rule](https://github.com/raywenderlich/swift-style-guide#type-inference), please remove type as it follows:
``` let date = Date()```

Reply:
Done.